src/OpenLoco/Paint/PaintEntity.h

```diff
@@ -1,11 +1,7 @@
 #pragma once
+#include "../Map/Map.hpp"
```

Comment:
Do we really need to include this header in this small one?

Reply:
I either include Map.hpp which already defines the types or I need to include Vector.hpp and define it again. Forward declaring the vector template is a bit ugly honestly.